frontend/src/components/OpeningHours/index.jsx

```diff
@@ -0,0 +1,72 @@
+import PropTypes from 'prop-types';
+import styles from './OpeningHours.module.scss';
+import { useTranslation } from 'react-i18next';
+
+const OpeningHours = function OpeningHours({
+  children,
+},
+) {
+  const { t } = useTranslation('openingHours');
+
+  const dateByDay = {
+    0: [],
+    1: [],
+    2: [],
+    3: [],
+    4: [],
+    5: [],
+    6: [],
+  };
+
+  const dayNumberToString = {
+    0: t('sunday'),
+    1: t('monday'),
+    2: t('tuesday'),
+    3: t('wenesday'),
+    4: t('thursday'),
+    5: t('friday'),
+    6: t('saturday'),
```

Comment:
Ça tu peux le mettre dans le `base.json` comme ça va etre commun a toute l'app

```suggestion
    0: t('weekDays.sunday', { ns: 'base' }),
    1: t('weekDays.monday', { ns: 'base' }),
    2: t('weekDays.tuesday', { ns: 'base' }),
    3: t('weekDays.wenesday', { ns: 'base' }),
    4: t('weekDays.thursday', { ns: 'base' }),
    5: t('weekDays.friday', { ns: 'base' }),
    6: t('weekDays.saturday', { ns: 'base' }),
```

Et ton `base.json` resemblera a ça
```json
{
  "weekDays": {
    "monday": "Lundi"
  }
}
```